public/app/core/components/OptionsUI/units.tsx

```diff
@@ -14,7 +14,12 @@ export function UnitValueEditor({ value, onChange, item }: Props) {
         <span className={styles.first}>
           <UnitPicker value={value} onChange={onChange} />
         </span>
-        <IconButton ariaLabel="clear unit selection" name="times" onClick={() => onChange(undefined)} />
+        <IconButton
+          ariaLabel="clear unit selection"
+          name="times"
+          onClick={() => onChange(undefined)}
+          tooltip="Clear unit section"
```

Comment:
```suggestion
          tooltip="Clear unit selection"
```

and does this mean we can remove the ariaLabel because it will just be the same as the tooltip?

Reply:
Yes, if there is no aria-label IconButton uses the tooltip for it.